scripts/build.sh

```diff
@@ -0,0 +1,12 @@
+openapi-generator-cli generate \
```

Comment:
This can be replace by node module
https://github.com/OpenAPITools/openapi-generator#17---npm

Reply:
@qtdan [Make it work, Make it right, Make it fast](https://thetombomb.com/posts/make-it-work-right-fast) ->First, I think we'll simply separate API spec to SDK (pulling from documentation source code) and still keep models in SDK.